### DISCUSSION TITLE
Downstream Systems and Talkgroup ID & Label Fields

### DISCUSSION THREAD

**Discussion by AvalancheAddict** [chuot/rdio-scanner#279] [Q&A]
Maybe I'm missing something, or maybe this will end up being more of a feature suggestion if it's not doable today, but is there a way to have the most "Downstream" system (so essentially the server that is taking in feeds from elsewhere) be the master of the Talkgroup & Label fields?

It would be nice if this is possible for a couple of different reasons.  First, if you, as the admin are in control of all of the systems involved (the upstream and downstream systems), you only need to maintain the names in one place, the final downstream server, rather than trying to maintain the names on the multiple systems feeding into it.  In addition, if you as the admin of the server are not in control of the systems feeding into it, you may want to use your own names, rather than someone else's.

In my case, I have new Talkgroups being detected on the upstream feeder system which then assigns the Talkgroup ID & Label fields to the number of the group.  I would like to change the ID / Label on only the downstream server only, as nobody is going to connect to the upstream system anyway, so there's no need to maintain the names there. 

When I do change it on the downstream "server", it only keeps the new name until a new audio file is fed into it.  As soon as that happens, it reverts back the the ID / Label names that the feeding system had (in my case, the default TG number).

Optimally a button on the downstream "server" that would allow the it to use it's own ID / Label fields whenever they are present, and to only use the ID / Label from upstream when there isn't already one set on the downstream server would probably be the answer.

Hopefully this makes sense, the upstream / downstream can get confusing.  That's why I generally say upstream "feeder" and downstream "server" to try to add some clarity.  - Thanks!

**Comment by chuot** [comment]
At this moment, the only way you can do it is by turning off the auto-populate option globally and on the affected system if it's on. You will have however to add talkgroups manually when new ones are discovered by the feeder instance.

**Reply by AvalancheAddict** [reply]
Thanks.  I figured this was the case, but wanted to make sure I wasn't missing something.  The auto-populate option is way too valuable to me, so I will just maintain it in both places, but perhaps an enhancement for you to consider some day.  Thanks again.
